src/fr/articles/svg-accessibles.md

```diff
@@ -1,142 +1,178 @@
+  &lt;span id="label" hidden&gt;Texte masqué&lt;/span&gt;
+  &lt;svg aria-hidden="true"&gt;&lt;!--...--&gt;&lt;/svg&gt;
+&lt;/a&gt;
 </code></pre>
 
-Enfin, ces deux solutions suivantes sont équivalentes et de toute façon à tester dans vos environnements de navigation (couples navigateur/aide technique) ciblés.
+La classe `visually-hidden` permet également de présenter le texte uniquement aux utilisateurs d'aide technique (masquage accessible). 
+La solution suivante est similaire à la précédente, mais à tester systématiquement dans vos environnements de navigation ciblés (couples navigateur/aide technique).
 
-<pre><code class="html">
+<pre><code class="html" title="html">
 &lt;button&gt;
-  &lt;svg focusable="false" aria-hidden="true"&gt;&lt;!--...--&gt;&lt;/svg&gt;
+  &lt;svg aria-hidden="true"&gt;&lt;!--...--&gt;&lt;/svg&gt;
   &lt;span class="visually-hidden"&gt;Rechercher&lt;/span&gt;
 &lt;/button&gt;
 
 &lt;a href="/Rechercher"&gt;
-  &lt;svg focusable="false" aria-hidden="true"&gt;&lt;!--...--&gt;&lt;/svg&gt;
+  &lt;svg aria-hidden="true"&gt;&lt;!--...--&gt;&lt;/svg&gt;
   &lt;span class="visually-hidden"&gt;Rechercher&lt;/span&gt;
 &lt;/a&gt;
 </code></pre>
 
-La class visually-hidden permet de cacher le texte sauf aux utilisateurs d’aide technique (masquage accessible).
+Il est aussi possible d'utiliser l'attribut `aria-label` :
 
-ou
-
-<pre><code class="html">
+<pre><code class="html" title="html">
 &lt;button aria-label="Rechercher"&gt;
-  &lt;svg focusable="false" aria-hidden="true"&gt;&lt;!--...--&gt;&lt;/svg&gt;
+  &lt;svg aria-hidden="true"&gt;&lt;!--...--&gt;&lt;/svg&gt;
 &lt;/button&gt;
 
 &lt;a href="/Rechercher" aria-label="Rechercher"&gt;
-  &lt;svg focusable="false" aria-hidden="true"&gt;&lt;!--...--&gt;&lt;/svg&gt;
+  &lt;svg aria-hidden="true"&gt;&lt;!--...--&gt;&lt;/svg&gt;
 &lt;/a&gt;
-
-ou, mais beaucoup plus risqué…
-&lt;button&gt;
-  &lt;svg aria-labelledby="rechercher-icon-title" focusable="false" role=”img”&gt;
-    &lt;title id="rechercher-icon-title"&gt;Rechercher&lt;/title&gt;
-    &lt;!--...--&gt;
-  &lt;/svg&gt;
-&lt;/button&gt;
 </code></pre>
 
-## Dernier mots…
+## Dernier mot…
 
-En résumé, un seul conseil pour du SVG accessible, ce serait TESTER et tester encore !
+En résumé, s'il ne fallait retenir qu'un seul conseil pour du SVG accessible, ce serait TESTER et tester encore !
 
-## Webographie (articles qui m’ont bien, bien aidés)
+## Webographie (articles qui m'ont bien, bien aidés)
 
-- https://www.deque.com/blog/creating-accessible-svgs/ 
-- https://weboverhauls.github.io/demos/svg/ 
-- https://www.scottohara.me/blog/2019/05/22/contextual-images-svgs-and-a11y.html 
-- https://www.slideshare.net/webaxe/svg-icons-and-screen-reader-accessibility 
-- https://github.com/JeremiePat/svg-accessible/blob/master/slides.md 
-- https://www.sarasoueidan.com/blog/accessible-icon-buttons/ 
+<a href="https://www.deque.com/blog/creating-accessible-svgs/" hreflang="en">Creating Accessible SVGs - Deque</a>
```

Comment:
Just a suggestion but it could be good to add `(en)` in the visible name to keep it consistent (see for instance: https://a11y-guidelines.orange.com/fr/articles/veille-octobre-novembre-2024/).
*Just made it on the first line but of course it would be great to add after each link in english.